src/fssrgen/FsSrGen.fsproj

```diff
@@ -1,60 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
they are being built for the .net framework
![](http://i.imgur.com/swve8va.png)